Eatery Blue/UI/HomeScreen/HomeViewController.swift

```diff
@@ -219,6 +253,25 @@ extension HomeViewController: UITableViewDataSource {
             let container = ContainerView(content: label)
             container.layoutMargins = Constants.labelLayoutMargins
 
+            let cell = ClearTableViewCell(content: container)
+            cell.selectionStyle = .none
+            return cell
+        case .statusLabel(status: let status):
```

Comment:
```suggestion
        case .statusLabel(let status):
```
I believe that should work. Change it for line 248 above as well.